src/stepper/stepper.ts

```diff
@@ -3361,6 +3369,10 @@ export function getEvaluationSteps(
       [],
       'Start of evaluation'
     ]
+    /**
+     * push the content into frontend for showing contents in the
+     * this comment may not be correct, please correct it if you find any mismatch in description
+     */
```

Comment:
This comment "this comment may not be correct" is quite strange. Are you referring to the one above: "push the content..."?